stable/insights-agent/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart to run the Fairwinds Insights agent
 name: insights-agent
-version: 1.16.0
-appVersion: 3.2.0
+version: 1.16.1
+appVersion: 6.5.1
```

Comment:
Is this version jump (3.2.0 to 6.5.1) correct?

Reply:
Yes from RB: https://fairwindsops.slack.com/archives/C01AJTQA9MK/p1637765776051700?thread_ts=1637763807.047800&cid=C01AJTQA9MK